### DISCUSSION TITLE
Bluetooth disconnected

### DISCUSSION THREAD

**Discussion by TheBinaryCoder0** [blueman-project/blueman#2455] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

My bluetooth device is connected and after 1 or 2 seconds it gets disconnected automatically and this process runs on a loop
kindly resolve my issue in this

**Comment by adriandelgg** [comment]
Part 6.4.2 of this Arch Wiki article may help your issue.
https://wiki.archlinux.org/title/bluetooth
